home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,33 @@
+# Local Library
+
+  
+{% hint style="info" %}  
+"iVP Library" will not show up if the folder is empty, and will show up as soon as items are added to it.  
+The folders name can be changed at any time.  
+Supported file-types include Images, .plan and 3D-Models.  
+{% endhint %}
+
+You can import your own files/objects from your PC into iVP by using the "Import" option.  
+
+![](.gitbook/assets/3d-import-file.png)
+
+When importing your file, you have the option to change its parameters, as well as the option to add it to your library.  
+
+![](.gitbook/assets/3d-import-addtolibrary.png)  
+
+Once you have added an item to your library, the folder "iVP Library" will appear at the bottom of the library window.
+From now on every item that you import, and chose to add to your library, can be found inside the "iVP Library" folder.  
+
+The Order of your items can only be changed inside the folder, **not** inside iVP-Planning.
+
+![](.gitbook/assets/3d-import-library.png)  
+
+Your "iVP Library" folder will be created in the default iVP-Directory.
+You can click on the "Open" button next to any object inside the fodler to open the directory directly.
+
+![](.gitbook/assets/3d-import-openobject.png)  
+
+The directory of your library can be changed manually in the settings.  
+Simply click on the "Local library path" option, and create a folder.
```

Comment:
Same here, give a bit more context. Explain where exactly you can find this option (which tab of the settings panel?), what to actually click (the button with the 3 dots next to the path) and what to do next.